### PR TITLE
merge/back-merge-nanny-arc-into-develop

### PR DESCRIPTION
**Pull request by Thomas-95IS**
## Description

Merge development from Nanny ARC back into `develop`.

Whilst a smoke test appears ok, there were a fair few conflicts and, given our non-exhaustive test coverage, I can't guarantee _all_ the functionality still works.

If the reviewer(s) can look for anything suspect or, better still, checkout the branch locally to have a play around, that'd be grand.

## Todo's before PR

- [x] Rebase from develop
- [ ] Unit tests passed (`make test`)
- [x] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates been checked with relevant user-story

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assigneses (those who'll merge)
- [x] Specified labels
- [x] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [x] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: arc_application/views/nanny_views/nanny_contact_details.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def create_context(self, application_id):
+        """
+        Creates the context dictionary for this view.
+        :param application_id: Reviewed application's id.
+        :return: Context dictionary.
+        """
+
+        # Get nanny information
+        identity_actions = IdentityGatewayActions()
+        contact_details = identity_actions.read('user',
```
Comment (ricaduchateau) [inline arc_application/views/nanny_views/nanny_contact_details.py]: The read method is not working with the provided parameters.

Comment (LRydeard) [reply arc_application/views/nanny_views/nanny_contact_details.py]: This issue is being caused by the Informed-CD-Scripts docker compose for ARC having incorrect parameters compared to the Nanny docker compose file.
This should not be an issue on the environments.

Comment (ricaduchateau) [reply arc_application/views/nanny_views/nanny_contact_details.py]: As discussed with @LRydeard, the ARC docker compose file has been updated in the OFS-MORE-DevOps-Tooling repository. Please see PR #83.

**Comment by LRydeard** [comment]
rerun jenkins build

**Pull request merged**